test-files/o2s_tds40.osm

```diff
@@ -1,71 +1,71 @@
+    <node visible="true" id="-11" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4294583412699993" lon="153.1629753005499879"/>
+    <node visible="true" id="-10" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4296521284900017" lon="153.1658720862899941"/>
+    <node visible="true" id="-9" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4273460403400016" lon="153.1640374553199990"/>
+    <node visible="true" id="-8" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4257084971699996" lon="153.1602394473599986">
+        <tag k="source:ingest:datetime" v="2020-11-13T21:04:23.704Z"/>
         <tag k="papa" v="smurf"/>
-        <tag k="source" v="cartoon"/>
-        <tag k="uuid" v="{bfbf2946-4342-444c-9926-1477c7bcce05}"/>
         <tag k="error:circular" v="15"/>
+        <tag k="uuid" v="{bfbf2946-4342-444c-9926-1477c7bcce05}"/>
+        <tag k="source" v="cartoon"/>
     </node>
-    <node visible="true" id="-7" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4270165820404834" lon="153.1617843997499904"/>
-    <node visible="true" id="-6" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4250108253499469" lon="153.1616663825500098"/>
-    <node visible="true" id="-5" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4249429972399419" lon="153.1638657939400048"/>
-    <node visible="true" id="-4" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4272491313805595" lon="153.1657004249099998"/>
-    <node visible="true" id="-3" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4270553402905044" lon="153.1628036391699936"/>
-    <node visible="true" id="-1" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4279564662007758" lon="153.1603145492099998">
-        <tag k="source:ingest:datetime" v="2017-12-18T20:34:25.107Z"/>
-        <tag k="building" v="hut"/>
+    <node visible="true" id="-7" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4270165955699987" lon="153.1617843997499904"/>
+    <node visible="true" id="-6" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4250108388799987" lon="153.1616663825500098"/>
+    <node visible="true" id="-5" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4249430107700007" lon="153.1638657939400048"/>
+    <node visible="true" id="-4" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4272491449100002" lon="153.1657004249099998"/>
+    <node visible="true" id="-3" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4270553538199984" lon="153.1628036391699936"/>
+    <node visible="true" id="-1" timestamp="1970-01-01T00:00:00Z" version="1" lat="-25.4279564797299997" lon="153.1603145492099998">
+        <tag k="source:ingest:datetime" v="2020-11-13T21:04:23.698Z"/>
+        <tag k="error:circular" v="15"/>
         <tag k="uuid" v="{1514e475-0a8b-450d-8138-af8623cbdd56}"/>
-        <tag k="source" v="tdsv40:hut_p"/>
```

Comment:
For some reason, the "source" tags were not being preserved.
I.e. "me" got wiped and was replaced with the standard  <spec>:<file name>